setup.py

```diff
@@ -364,9 +364,9 @@ def parse_jsargs():
         sys.argv.remove('--build_js')
 
     elif '--install_js' in sys.argv:
-        if not installing:
-            print("Error: Option '--install_js' only valid with 'install', 'develop' or 'sdist', exiting.")
-            sys.exit(1)
+        #if not installing:
+        #    print("Error: Option '--install_js' only valid with 'install', 'develop' or 'sdist', exiting.")
+        #    sys.exit(1)
```

Comment:
The code I commented prevented the use of only doing `setup.py --install_js`; you could not install js without also installing bokeh-the-package. Having the option to only install js is beneficial, IMO, because its faster and it will always work (i.e. on system Python you wont have permission unless its run under sudo).

I should probbaly *remove* the code rather than comment it though..